.gitlab-ci.yml

```diff
@@ -104,7 +104,7 @@ lv-cuda:
 # -- PETSc with CUDA (minimal)
     - export PETSC_DIR=/home/jeth8984/petsc PETSC_ARCH=cuda-O && git -C $PETSC_DIR describe
     - echo "-------------- PETSc ---------------" && make -C $PETSC_DIR info
-    - make -k -j$NPROC_GPU BACKENDS="$BACKENDS_GPU" junit search="petsc fluids solids"
```

Comment:
Fluids and Solids are fairly resource heavy - I think we might not be able to test them here

Reply:
What metric are you using? What would be the marginal cost to do one run of each in this environment?

Reply:
Before I dropped these two I got a bunch of`CUDA_ERROR_LAUNCH_OUT_OF_RESOURCES` error messages. It might be easier to adjust this over in Leila's branch where we aren't constantly pulling vectors back and forth between host and device.